src/xdocs/config_javadoc.xml

```diff
@@ -649,6 +649,13 @@ public int checkReturnTag(final int aTagIndex,
               <td><code>false</code></td>
               <td>6.0</td>
             </tr>
+            <tr>
+              <td>validateParameterOrder</td>
+              <td>Control whether to validate order of <code>param</code> tags.</td>
+              <td><a href="property_types.html#boolean">Boolean</a></td>
+              <td><code>false</code></td>
+              <td>8.4</td>
```

Comment:
This is being released in 8.4? Last I checked we are working towards 8.31 .

Reply:
I saw that the last release of check style on GitHub was 8.4. So I wrote 8.4. What should I write instead ?

Reply:
https://github.com/checkstyle/checkstyle/releases
We just release 8.31 . So next release is 8.32 . I am not sure where you saw 8.4 .